modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1218,136 +1218,6 @@ The query contains an aggregation function that skips null values.
 ======
 =====
 
-[#_unsupported_notifications]
-== `UNSUPPORTED` notifications
-
-Unsupported notifications are returned when the query or command is trying to use features that are not supported by the current system or using experimental features that should not be used in production.
-
-[#_neo_clientnotification_statement_runtimeunsupportedwarning]
-=== Unsupported runtime
-
-.Notification details
-[cols="<1s,<4"]
-|===
-|Neo4j  code
-m|Neo.ClientNotification.Statement.RuntimeUnsupportedWarning
-|Title
-a|This query is not supported by the chosen runtime.
-|Description
-|Selected runtime is unsupported for this query, please use a different runtime instead or fallback to default.
-(`%s`)
-|Category
-m|UNSUPPORTED
-|GQLSTATUS code
-m|01N40
```

Comment:
Isn't the removal of this code considered a breaking change?

Reply:
I am fine if that's ok with you. I just questioned whether 01N40 has been deprecated for a year.

Reply:
Oh it looks like I have removed two examples but only intended to remove one of them. Don't know why... I will fix it